src/test/regress/sql/pg15.sql

```diff
@@ -799,6 +799,81 @@ CREATE TABLE set_on_default_test_referencing(
 
 SELECT 1 FROM citus_remove_node('localhost', :master_port);
```

Comment:
How about deleting this line, along with the `citus_add_node` you introduced?

Reply:
We need to add the coordinator in order to have a Citus local table. However, the previous test also adds and removes the coordinator. So I will remove the `citus_add_node` that I introduced, along with the `citus_remove_node` from the previous test.